bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/handler/SatelSystemHandler.java

```diff
@@ -79,7 +79,9 @@ public void incomingEvent(IntegraStatusEvent event) {
         if (getThingConfig().isCommandOnly()) {
             return;
         }
-        updateState(CHANNEL_DATE_TIME, new DateTimeType(event.getIntegraTime().atZone(getBridgeHandler().getZoneId())));
+        event.getIntegraTime().ifPresent(dt -> {
+            updateState(CHANNEL_DATE_TIME, new DateTimeType(dt.atZone(getBridgeHandler().getZoneId())));
```

Comment:
Should the date time channel be set to undef if it isn't present?

Reply:
It could, but I don't think it is necessary. You cannot enter incorrect date or time in the panel I believe and therefore it cannot change from correct to incorrect. This is the only case when it should be reset to undef. 
For the sake of completeness I can add it if you think it does make sense.

Reply:
Please add it

Reply:
Done.